crates/cairo-profiler/src/trace_reader/function_trace_builder.rs

```diff
@@ -46,13 +49,15 @@ enum MaybeSierraStatementIndex {
 }
 
 /// Collects profiling info of the current run using the trace.
+#[allow(clippy::too_many_lines)]
```

Comment:
No longer needed (checked it)

Reply:
yeah me too :D 

```
warning: this function has too many lines (113/100)
   --> crates/cairo-profiler/src/trace_reader/function_trace_builder.rs:52:1
    |
52  | / pub fn collect_function_level_profiling_info(
53  | |     trace: &[TraceEntry],
54  | |     program: &Program,
55  | |     casm_debug_info: &CairoProgramDebugInfo,
...   |
211 | |     }
212 | | }
    | |_^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#too_many_lines
    = note: `-W clippy::too-many-lines` implied by `-W clippy::pedantic`
    = help: to override `-W clippy::pedantic` add `#[allow(clippy::too_many_lines)]`

```

Reply:
Mb, run `cargo clippy` instead of `cargo lint` 🤦